### ISSUE TITLE
Accessibility problems with polls

### ISSUE THREAD

**Issue by charlag** [tuskyapp/Tusky#1287]
- Polls are not announced by the screen reader
- No way to vote for polls with a screen reader

It shouldn't be too hard to add it but the problem is that translations must be updated accordingly. Not sure how to solve this yet, any ideas @connyduck ?

**Comment by charlag** [comment]
Maybe I can smash poll description inside the content, not sure yet

**Comment by charlag** [comment]
After testing I see that non-finished polls can actually be acted upon but finished ones are not announced at all
